include/ncpp/Plane.hh

```diff
@@ -964,11 +964,13 @@ namespace ncpp
 
 		ncplane* duplicate_plane (const Plane &other, void *opaque)
 		{
+			if (other.is_stdplane)
+				throw init_error ("Standard plane cannot be duplicated");
```

Comment:
So the standard plane *can* be duplicated, just the result is not the standard plane.

I.e. if I write "frooglegobble" to the center of the standard plane, and then run `ncplane_dup()` on that, I get a plane which is the same size as the standard plane, has the same content ("frooglegobble" in the middle), but this plane (unlike the standard plane) I can move, resize, and destroy.

Effectively, `ncplane_dup()` needs no special case for the standard plane. Here's the C source:

```c
ncplane* ncplane_dup(ncplane* n, void* opaque){
  int dimy = n->leny;
  int dimx = n->lenx;
  int aty = n->absy;
  int atx = n->absx;
  int y = n->y;
  int x = n->x;
  uint32_t attr = ncplane_attr(n);
  uint64_t chan = ncplane_channels(n);
  ncplane* newn = ncplane_new(n->nc, dimy, dimx, aty, atx, opaque);
  if(newn){
    if(egcpool_dup(&newn->pool, &n->pool)){
      ncplane_destroy(newn);
    }else{
      ncplane_cursor_move_yx_locked(newn, y, x); // FIXME what about error?
      n->attrword = attr;
      n->channels = chan;
      ncplane_move_above_unsafe(newn, n);
      memcpy(newn->fb, n->fb, sizeof(*n->fb) * dimy * dimy);
    }
  }
  return newn;
}
```

the standard plane can go through there just fine. The result is an exact duplicate of the standard plane, save a copy of its deep structures, but this plane is not *the* (there can only be one!) standard plane.

The difference comes down to how in my C, there is no "am i the standard plane?" field in `ncplane`. There's simply a `notcurses` struct which has a single `->stdscr` member.